src/ids.rs

```diff
@@ -41,6 +45,30 @@ impl PyIds {
         if let Some(constant) = self.constants.get(name) {
             return Ok(constant.to_object(py));
         }
+
+        // Support for for ids.{Struct Definition} information
+        // Example: ids.DictAccess
+        let mut types_set = HashSet::new();
+        for key in self.struct_types.keys() {
+            types_set.insert(key.split('.').last());
```

Comment:
```suggestion
            types_set.insert(key.rsplit('.').next().ok_or(...)?);
```

Reply:
Please add a `TODO` comment, we should eventually move this logic to the program loading step so this one is just merging the `HashMap`s.

Reply:
I created an issue
https://github.com/lambdaclass/cairo-rs/issues/532